### PR TITLE
Remove `donate` app from RTD itself

### PR DESCRIPTION
**Pull request by ericholscher**
This splits out the `donate` app from RTD, so that we don't ship our ad code to all the OSS users who don't care about it. 

The actual changes are at the bottom, just some URL adjustments.

### CODE REVIEW

File: readthedocs/urls.py [hunk 1]
```diff
@@ -44,6 +44,7 @@
     url(r'^accounts/', include('readthedocs.profiles.urls.private')),
     url(r'^accounts/', include('allauth.urls')),
     url(r'^notifications/', include('readthedocs.notifications.urls')),
+    url(r'^accounts/gold/', include('readthedocs.gold.urls')),
```
Comment (ericholscher) [inline readthedocs/urls.py]: Other question is do we move Gold out as well, and keep it gated in a similar way?

Comment (agjohnson) [reply readthedocs/urls.py]: Eventually, +1 from me. We can address this separately. Feel free to open a ticket to track the work.

File: readthedocs/donate/migrations/0005_add-geo-filters.py [hunk 2]
```diff
@@ -1,32 +0,0 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-
-from __future__ import absolute_import
-from django.db import models, migrations
-import django_countries.fields
```
Comment (agjohnson) [inline readthedocs/donate/migrations/0005_add-geo-filters.py]: Can we remove this dependency? We can probably move this to the ext repo if so.

File: readthedocs/donate/signals.py [hunk 3]
```diff
@@ -1,264 +0,0 @@
...
-from readthedocs.donate.constants import INCLUDE, EXCLUDE
-from readthedocs.donate.utils import offer_promo
-
-
-log = logging.getLogger(__name__)
-
-PROMO_GEO_PATH = getattr(settings, 'PROMO_GEO_PATH', None)
-
-if PROMO_GEO_PATH:
-    import geoip2.database  # noqa
```
Comment (agjohnson) [inline readthedocs/donate/signals.py]: Same on this import

Comment (ericholscher) [reply readthedocs/donate/signals.py]: Believe that is installed via `ops`.

File: readthedocs/donate/migrations/0005_add-geo-filters.py (hunk 2 above)
Comment (ericholscher) [reply readthedocs/donate/migrations/0005_add-geo-filters.py]: Yep.

**Comment by ericholscher** [comment]
🎆

**Pull request merged**